recipes/breadability/LICENSE.rst

```diff
@@ -0,0 +1,10 @@
+Copyright (c) 2013 Rick Harding and contributors
```

Comment:
The `breadability` sdist already contains this file so you should remove it from this PR.

Reply:
@dopplershift done, thank you.